### DISCUSSION TITLE
Disabling i3-bar to use Polybar

### DISCUSSION THREAD

**Discussion by mauromotion** [regolith-linux/regolith-desktop#824] [Q&A] [answered]
I've been searching everywhere but it seems like since Regolith version 2 the old solutions aren't usable. I believe this new version should be more modular? 
I can't see any specific package for the i3-bar itself.

How can I disable i3-bar completely then?

**Comment by cfsmp3** [comment, marked as answer]
> How can I disable i3-bar completely then?

```
/usr/share/regolith/i3/config.d/70_bar
```

**Reply by mauromotion** [reply]
Would you mind sharing some more information? I tried everything with that file with no luck. 

As for the widgets I copied the whole dir and files in my .config and then removed 70_bar. No luck.

I tried commenting inside the file. No luck.

What am I supposed to do?

EDIT: Wait, I did it. I had to comment both files.

**Reply by cfsmp3** [reply]
That file points to the program that creates the bar.

You can see that here:

```
set_from_resource $wm.bar.status_command wm.bar.status_command i3xrocks -u ~/.config/regolith2/i3xrocks/conf.d -d /usr/share/i3xrocks/conf.d
```
You can replace i3xrocks with whatever you want. I use i3status-rs.

**Reply by mauromotion** [reply]
That's brilliant, thanks!

**Comment by anamyk** [comment]
Could you share your exact command please? I cant get it work
